wire-ios/Wire-iOS.xcodeproj/xcshareddata/xcschemes/Wire-iOS.xcscheme

```diff
@@ -20,6 +20,244 @@
                ReferencedContainer = "container:Wire-iOS.xcodeproj">
             </BuildableReference>
          </BuildActionEntry>
+         <BuildActionEntry
```

Comment:
This is the only change that might cause trouble. In Wire-iOS scheme, I am marking all test targets to be compiled for the test build action. This would slow down running local tests. An alternative would be creating a new scheme specifically for selective testing. I was not bold enough to start managing schemes in the project.